docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,217 @@
+
+You can specify local browsers by using paths or [aliases](/testcafe/documentation/using-testcafe/common-concepts/browser-aliases). 
+A list of all the available aliases can be obtained by the [--list-browsers](#b) command.
+
+The following example demonstrates how to run a test in several browsers. 
+The browsers are specified differently: one by using the alias, the other by using the path.
+
+```sh
+testcafe chrome,/applications/safari.app /tests/sample-fixture.js
+```
+
+To run tests against **all installed browsers**, use the `all` alias.
+
+```sh
+testcafe all /tests/sample-fixture.js
+```
+
+### <a class="anchor" name="remotes"></a>Remote Browsers
+
+To run tests on a remote browser, specify `remote` as a browser alias. 
+
+```sh
+testcafe remote /tests/sample-fixture.js
+```
+
+If you want to connect multiple browsers,
+prefix alias with the number of browsers you want to connect. For example, if you need to use three remote browsers, specify *3remote*.
+
+```sh
+testcafe 3remote /tests/sample-fixture.js
+```
+
+TestCafe will provide URLs that you need to open in required browsers on your remote device. 
+
+You can also use the [--qr-code](#qr-code) option to display QR-codes that represent the same URLs. 
+Scan the QR-codes by a device on which you are going to test the application.
+As a result, the browsers will be connected to TestCafe and tests will start.
+
+## File Path/Glob Pattern
+
+The `file-or-glob ...` argument specifies files or directories(separated by a space) to run tests from.
+
+For example, this command runs all tests located in the `/my-tests` directory.
+
+```sh
+testcafe ie /my-tests
+```
+
+The following command runs tests from the certain fixture files.
+
+```sh
+testcafe ie /tests/fixture1.js /tests/fixture2.js
+```
+
+You can also use [globbing patterns](https://github.com/isaacs/node-glob#glob-primer) to specify a set of files. 
+For example, this command runs tests from files that matches `/tests/*page*`, namely `/tests/example-page.js` and `/tests/main-page.js`.
+
+```sh
+testcafe ie /tests/*page*
+```
+
+If you do not specify any file or directory, TestCafe will run tests from `/test` or `/tests` directories.
+
+## Options
+
+### -h, --help  
+Displays usage information for commands.
+
+```sh
+testcafe --help
+```
+
+### <a class="anchor" name="b"></a>-b, --list-browsers 
+Lists [aliases] of the browsers available on this computer.
+
+```sh
+testcafe --list-browsers
+```
+
+### -r \<name\>, --reporter \<name\>  
+Specifies a built-in or custom reporter that will be used to generate test reports. 
+The available built-in reporters are `json`, `list`, `minimal`, `spec`, `xunit`. By default, the `spec` reporter is used.
+
+The following command tells TestCafe to run the tests in all available browsers and generate a report in the xunit format.
+
+```sh
+testcafe all /tests/sample-fixture.js -r xunit
+```
+
+The following command tells TestCafe generate a test report by using the custom plugin reporter.
```

Comment:
The topic about custom reporters doesn't exist yet. After creating it, we'll add a link.